autoload/asyncomplete.vim

```diff
@@ -244,27 +238,15 @@ function! s:get_active_sources_for_buffer() abort
     return l:active_sources
 endfunction
 
-if exists('*matchstrpos')
-    function! s:matchstrpos(expr, pattern) abort
-        return matchstrpos(a:expr, a:pattern)
-    endfunction
-else
-    function! s:matchstrpos(expr, pattern) abort
-        return [matchstr(a:expr, a:pattern), match(a:expr, a:pattern), matchend(a:expr, a:pattern)]
-    endfunction
-endif
-
```

Comment:
Not needed as `matchstrpos` was added in [Patch 7.4.1685](http://ftp.vim.org/vim/patches/7.4/7.4.1685) so each version that has async support has this function.

Reply:
Old neovim doesn’t have it so I wouldn’t remove it yet.